src/components/new-request/search.vue

```diff
@@ -315,40 +365,20 @@ export default class Search extends Mixins(CommonMixin) {
     return (this.getRequestActionCd === NrRequestActionCodes.NEW_BUSINESS && this.request?.group === 1)
   }
 
-  get isContinuationIn (): boolean {
-    return (this.getRequestActionCd === NrRequestActionCodes.MOVE)
-  }
-
-  get isAlteration (): boolean {
-    return (this.getRequestActionCd === NrRequestActionCodes.CONVERSION)
-  }
-
-  get isAmalgamation (): boolean {
-    return (this.getRequestActionCd === NrRequestActionCodes.AMALGAMATE)
-  }
-
-  get isChangeName (): boolean {
-    return (this.getRequestActionCd === NrRequestActionCodes.CHANGE_NAME)
-  }
-
-  get isRestoration (): boolean {
-    return (this.getRequestActionCd === NrRequestActionCodes.RESTORE)
-  }
-
   get showJurisdiction (): boolean {
     // if (this.isAmalgamation) return true // *** FUTURE
     if (this.isNewXproBusiness) return true
     return false
   }
 
   get showEntityType (): boolean {
-    if (this.getLocation) return true
+    if (this.getLocation && !this.isFederal) return true
     return false
   }
 
   get showBusinessLookup () {
     if (this.isChangeName) return true
-    if (this.isAlteration) return true
+    if (this.isConversion) return true
```

Comment:
A conversion is an alteration, but the NR Request Action Code is CNV so I renamed this getter accordingly.